ui/src/app/(index)/page.tsx

```diff
@@ -1,6 +1,74 @@
-const Home = () => {
+import Image from 'next/image';
+import BeforeLogin from '@/app/(index)/_components/BeforeLogin';
+import AfterLogin from '@/app/(index)/_components/AfterLogin';
+import {getCurrentUser} from '@/access/AuthenticationService';
+import Role from '@/access/Role';
+import LoginButton from '@/app/(index)/_components/LoginButton';
+import {getAnnouncements, getNumberOfGroupings, getNumberOfMemberships} from '@/services/GroupingsApiService';
+import Announcement from '@/app/(index)/_components/Announcement';
+
+const Home = async () => {
+    const [currentUser, numberOfGroupings, numberOfMemberships, announcements] = await Promise.all([
+        getCurrentUser(),
+        getNumberOfGroupings(),
+        getNumberOfMemberships(),
+        getAnnouncements()
+    ]);
+
+    const activeAnnouncements = () => {
+        if (!announcements) {
+            return []
+        }
+        return announcements.announcements
+            .filter((announcement) => announcement.state === 'Active')
+            .map((announcement) => announcement.message)
+    }
+
     return (
-        null
+        <main>
+            <div className="container mt-5 mb-5">
+                {activeAnnouncements().map((announcement: string, index: number) => (
+                    <Announcement announcement={announcement} key={index}/>
+                ))}
```

Comment:
I think it would look better if we put this inside an <Announcements /> component. Thanks!